src/pages/Share/index.js

```diff
@@ -17,35 +17,15 @@ import { TwitterTimelineEmbed } from 'react-twitter-embed';
 const StarMediaSection = () => {
   const classes = useStyles();
 
-  const CardSection = ({ image, title, cardContent, link = '' }) => {
+  const GitHubCard = () => {
```

Comment:
Was there a reason to create a new component specifically for the GitHub card?

Reply:
The CardSection component uses a CardMedia section that is not compatible with the GitHubButton component. I have gotten rid of the GitHubCard component and put the code for creating it back in the grid where it will be displayed.